inventory.go

```diff
@@ -41,104 +76,167 @@ type InventoryAppStats struct {
 	params := url.Values{
-		"start": {strconv.FormatUint(uint64(start), 10)},
-	}
-	if tradableOnly {
-		params.Set("trading", "1")
+		"l":             {string(lang)},
+		"start_assetid": {strconv.FormatUint(startAssetID, 10)},
 	}
 
-	resp, err := session.client.Get(fmt.Sprintf("https://steamcommunity.com/profiles/%d/inventory/json/%d/%d/?", sid, appID, contextID) + params.Encode())
+	requestURL := fmt.Sprintf("http://steamcommunity.com/inventory/%d/%d/%d", sid, appID, contextID)
+
+	resp, err := session.client.Get(requestURL + "?" + params.Encode())
 	if err != nil {
-		return 0, err
+		return false, 0, err
+	}
+
+	type Asset struct {
+		AppID      uint64 `json:"appid,string"`
+		ContextID  uint64 `json:"contextid,string"`
+		AssetID    uint64 `json:"assetid,string"`
+		ClassID    uint64 `json:"classid,string"`
+		InstanceID uint64 `json:"instanceid,string"`
+		Amount     uint64 `json:"amount,string"`
+	}
+
+	type DescriptionsPart struct {
+		Type  string `json:"type"`
+		Value string `json:"value"`
+		Color string `json:"color"`
+	}
+
+	type Description struct {
+		AppID                     uint64             `json:"appid"`
+		ClassID                   uint64             `json:"classid,string"`
+		InstanceID                uint64             `json:"instanceid,string"`
+		Currency                  int                `json:"currency"`
+		BackgroundColor           string             `json:"background_color"`
+		IconURL                   string             `json:"icon_url"`
+		IconURLLarge              string             `json:"icon_url_large"`
+		Descriptions              []DescriptionsPart `json:"descriptions"`
+		Tradable                  int                `json:"tradable"`
+		Name                      string             `json:"name"`
+		NameColor                 string             `json:"name_color"`
+		Type                      string             `json:"type"`
+		MarketName                string             `json:"market_name"`
+		MarketHashName            string             `json:"market_hash_name"`
+		Commodity                 int                `json:"commodity"`
+		MarketTradableRestriction int                `json:"market_tradable_restriction"`
+		Marketable                int                `json:"marketable"`
 	}
 
 	type Response struct {
-		Success      bool            `json:"success"`
-		ErrorMsg     string          `json:"Error"`
-		MoreStart    interface{}     `json:"more_start"` // This can be a bool or a number...
-		Inventory    json.RawMessage `json:"rgInventory"`
-		Descriptions json.RawMessage `json:"rgDescriptions"`
-		/* Missing: rgCurrency  */
+		Assets              []Asset       `json:"assets"`
+		Descriptions        []Description `json:"descriptions"`
+		Success             int           `json:"success"`
+		HasMore             int           `json:"more_items"`
+		LastAssetID         string        `json:"last_assetid"`
+		TotalInventoryCount int           `json:"total_inventory_count"`
```

Comment:
@asamy We can do it, but not with current realization. Now `items` array initialized in `GetInventory` method, but filled in `fetchInventory`, so on initialization time we don't know totalCount